mmv1/third_party/terraform/website/docs/r/bigquery_table.html.markdown

```diff
@@ -159,6 +165,11 @@ in Terraform state, a `terraform destroy` or `terraform apply` that would delete
     the form `{{project}}.{{location}}.{{connection_id}}`
     or `projects/{{project}}/locations/{{location}}/connections/{{connection_id}}`.
 
+    ~>**NOTE:** If you specify `connection_id`, Do not use
```

Comment:
Similar to the above feedback, what would you think about something like:

```suggestion
    ~>**NOTE:** If you set `external_data_configuration.connection_id`, the table schema must be specified using the top-level `schema` field [documented above](#schema).
```

(You'd also need to add an anchor for `schema`.)